dev.slimevr.SlimeVR.metainfo.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+dev.slimevr.SlimeVR.metainfo.xml by SlimeVR contributors
+
+To the extent possible under law, the person who associated CC0 with
+dev.slimevr.SlimeVR.metainfo.xml has waived all copyright and related or neighboring rights
+to dev.slimevr.SlimeVR.metainfo.xml.
+
+You should have received a copy of the CC0 legalcode along with this
+work.  If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+-->
+<component type="desktop-application">
+  <id>dev.slimevr.SlimeVR</id>
+
+  <name>SlimeVR</name>
+  <summary>An app for facilitating full-body tracking in virtual reality</summary>
+
+  <!-- CC0 so attribution is not required -->
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT OR Apache-2.0</project_license>
+
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://slimevr.dev/</url>
+  <url type="bugtracker">https://github.com/SlimeVR/SlimeVR-Server/issues</url>
+  <url type="faq">https://docs.slimevr.dev/server-setup/slimevr-setup.html</url>
+  <url type="vcs-browser">https://github.com/SlimeVR/SlimeVR-Server</url>
+  <recommends>
+    <display_length compare="ge">880</display_length>
+  </recommends>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+
+  <description>
+    <p>
+      <em>SlimeVR</em>
+      is a set of open hardware sensors and open source software that facilitates full-body
+      tracking (FBT) in virtual reality. With no base station required, SlimeVR makes wireless
+      VR FBT affordable and comfortable.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">dev.slimevr.SlimeVR.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://media.discordapp.net/attachments/818062236492759050/1044722428951134310/image.png</image>
```

Comment:
I don't like this linking to Discord, can we just put the image on GitHub in this repo and link to it instead?